docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md

```diff
@@ -1,19 +1,16 @@
 ---
-title: Deploy ML.NET Model to Azure Functions
+title: "How-To: Deploy ML.NET machine learning model to Azure Functions"
 description: Serve ML.NET sentiment analysis machine learning model for prediction over the internet using Azure Functions
-ms.date: 03/08/2019
+ms.date: 04/24/2019
+author: luisquintanilla
+ms.author: luquinta
 ms.custom: mvc,how-to
 #Customer intent: As a developer, I want to use my ML.NET Machine Learning model to make predictions through the internet using Azure Functions
 ---
 
-# How-To: Use ML.NET Model in Azure Functions
+# How-To: Deploy ML.NET machine learning model to Azure Functions
 
-This how-to shows how individual predictions can be made using a pre-built ML.NET machine learning model through the internet in a serverless environment such as Azure Functions.
-
-> [!NOTE]
-> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
-
-This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes).
+Learn how to deploy a pre-trained ML.NET machine learning model for predictions over the internet through an Azure Functions serverless environment.
```

Comment:
Over the internet --> in the cloud?? maybe

Reply:
#resolved